### PR TITLE
M3-5766: LKE UI refinements

### PR DESCRIPTION
**Pull request by HanaXu**
## Description 📝

- Tighten up spacing in the Kubernetes Cluster details page
- Refactoring and code clean up
- Added compactY to Button component and renamed compact to compactX
- Updated Storybook Button to include compactY

## Preview 📷

Before
image

After
image

## How to test 🧪
- Go to a Kubernetes cluster details page and check the changes
- Run Storybook locally (`yarn storybook`) and check the Button story

### CODE REVIEW

File: packages/manager/src/components/Button/button.stories.mdx [hunk 1]
```diff
@@ -127,13 +131,20 @@ export const Template = (args) => (
...
         description:
           'Removes the spacing to the left and right of the text for secondary buttons only.',
         table: {
           defaultValue: { summary: 'false' },
         },
       },
+      compactY: {
+        description:
+          'Removes the top and bottom spacing for secondary buttons only.',
+        table: {
```
Comment (dwiley-akamai) [inline packages/manager/src/components/Button/button.stories.mdx:144]: extremely minor, but maybe match up the descriptions of `compactX` and `compactY`: "Removes the spacing [...]" vs. "Removes the [...] spacing"

**Review by dwiley-akamai — COMMENTED** [review]
looks good, I did notice in mobile view that the "Delete Cluster" button isn't aligned as it is when the viewport is larger:

Screen Shot 2022-10-19 at 11 10 01 AM

**Comment by hana-linode** [comment]
@dwiley-akamai Running into trouble trying to align the Delete Cluster button for smaller viewports, will make a ticket to address that later

**Comment by hana-linode** [comment]
@dwiley-akamai spacing has been updated. Hesitant to lower it more as it would make the left and right spacing difference more obvious
image

**Review by dwiley-akamai — APPROVED** [review]

**Review by bnussman-akamai — COMMENTED** [review]
Spacing here looks off compared to the `After` you shared

Screen Shot 2022-10-26 at 11 27 47 AM

**Comment by hana-linode** [comment]
@bnussman-akamai hmm not seeing this, can you provide steps to reproduce? 🤔

**Comment by jdamore-linode** [comment]
Also seeing the increased spacing above the "HA Cluster", "Kubernetes Dashboard", etc. buttons. I think it's due to the flex positioning and can vary depending on the height of the nearby content:

Screen Shot 2022-10-28 at 1 01 11 PM

Setting that div's `justify-content` property to `flex-start` as opposed to `flex-end` seems to bring it a little closer to the desired position, but it may be a bit too close to the top in that case and could use some tweaking.

This is the only issue I'm seeing besides the "Delete Cluster" button alignment -- I can give this an approval if we want to handle it in a later effort too

**Review by jdamore-linode — APPROVED** [review]

**Pull request merged**